Snowflake.Data/Core/SFReusableChunk.cs

```diff
@@ -74,10 +81,10 @@ private class BlockResultData
             private static readonly int NULL_VALUE = -100;
             private int blockCount;
 
-            private static int blockLengthBits = 24;
+            private static int blockLengthBits = 23;

```

Comment:
why do we change this?

Reply:
To have two times smaller blocks. It is in par with jdbc: https://github.com/snowflakedb/snowflake-jdbc/blob/d26df3a367fbbde9082372e1d6f4a1c168ed31b9/src/main/java/net/snowflake/client/jdbc/JsonResultChunk.java#L449